[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e

xbpiao opened the pull request:
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e
(win32 support  android-ndk-r9d android-ndk-r10d android-ndk-r10e android-ndk-r11c android-ndk-r13b android-ndk-r14b)

xbpiao committed:
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e
(win32 support  android-ndk-r9d android-ndk-r10d android-ndk-r10e android-ndk-r11c android-ndk-r13b android-ndk-r14b)

xbpiao commented:
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e
(win32 support  android-ndk-r9d android-ndk-r10d android-ndk-r10e android-ndk-r11c android-ndk-r13b android-ndk-r14b)

macOS下android-ndk-r13b android-ndk-r14b生成时cocos2dx_physics3d.ini会出错其它正确，目前只能测试到这些，其它NDK版本未测试，但相比原来的版本兼容性已经好很多。

dumganhar commented on tools/tolua/cocos2dx_3d.ini:
```diff
@@ -7,10 +7,10 @@ prefix = cocos2dx_3d
 target_namespace = cc
 
-android_headers = -I%(androidndkdir)s/platforms/android-14/arch-arm/usr/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.7/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.7/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.8/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/4.8/include
+android_headers = -I%(androidndkdir)s/platforms/android-14/arch-arm/usr/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/include -I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/libs/armeabi-v7a/include -I%(androidndkdir)s/sources/cxx-stl/gnu ...
```
There is a duplicated of `-I%(androidndkdir)s/sources/cxx-stl/gnu-libstdc++/%(gnu_libstdc_version)s/libs/armeabi-v7a/include`.
The old one is using 4.7 and 4.8, but currently, it uses `gnu_libstdc_version` variable. So we could remove one.

dumganhar replied:
The same problem in other ini files.

xbpiao replied:
Thanks for the reminder that you are correct and now delete the duplicate configuration.

xbpiao committed:
[tolua]Lua bindings-generator support android-ndk-r9d android-ndk-r10d android-ndk-r10e
(win32 support  android-ndk-r9d android-ndk-r10d android-ndk-r10e android-ndk-r11c android-ndk-r13b android-ndk-r14b)
Remove duplicate configuration

xbpiao commented:
咦！感觉这样修改兼容性好很多的。

minggo commented:
Sorry, i may close it by mistook. And i can not reopen it. Could you please send another PR? Thanks.

dumganhar commented:
@xbpiao, v3.15 branch has been deleted. So this pull request was automatically closed.
Could you send it to v3 branch again? Thanks.

xbpiao commented:
@dumganhar @minggo 
#17817
